client/state/sites/plans/selectors.js

```diff
@@ -98,6 +118,32 @@ export function getPlanDiscountedRawPrice(
 }
 
 /**
+ * Returns a plan price before discount
+ *
+ * @param  {Object}  state         global state
+ * @param  {Number}  siteId        the site id
+ * @param  {String}  productSlug   the plan product slug
+ * @param  {Boolean} isMonthly     if true, returns monthly price
+ * @return {Number}                plan raw price
+ */
+export function getPlanRawPrice(
```

Comment:
+1 for distinguishing the two. Until you mentioned this here I didn't realize there were two separate `plans` branches of the state tree.

Not sure if eventually migrating to global selectors (#5954) is still on the table (@aduth?). That would also be a good case for naming these `getSitePlan*`